styleguide/derek/_partials/solutions/leadSplit.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Lead pattern -->
+<!--
+.lead class naming not used in alignment with typical pattern structure
+instead here .lead-wrapper is used to avoid having to override bootstrap base class
+ -->
+<div class="lead-wrapper">
+  <div class="lead-headerWrapper">
+    <h1 class="lead-header left">Title goes here</h1>
```

Comment:
Can we have a `lead-headerLeft` instead here? Additionally you may want to confirm this with Mike, because he mentioned something earlier about only ever using leads by themselves at the top of the page, and not inside mini panels like this. He may be okay with it, but I'd just recommend asking him about that.